src/net-utils.ts

```diff
@@ -39,6 +39,63 @@ async function isPortOpen(
   });
 }
 
+/**
+ * Repeatedly attempts to connect to a specified port until it becomes available
+ * or times out. Uses a shorter connection timeout (200ms) for each attempt
+ * while respecting the overall timeout period.
+ *
+ * @param port - The port number to check
+ * @param timeout - Maximum time in milliseconds to wait for the port to open
+ * (default: 1000ms)
+ * @returns A promise that resolves to true if the port opens within the timeout
+ * period, false otherwise
+ */
+export async function waitUntilServerPortIsAvailable(
+  port: number,
+  timeout: number = 1000
+): Promise<boolean> {
+  const portIsOpen: true | undefined = await retryUntilTimeout(
+    timeout,
+    async (): Promise<true> => {
+      if ((await isServerPortAvailable(port)) === true) {
+        return true;
+      } else {
+        // This callback needs to throw an error for the retry to happen.
+        throw new Error(`Port ${port} is not open yet`);
+      }
+    },
+    80
```

Comment:
What's this argument?

Reply:
That's how long to delay between retries. The previous default, 20ms, seemed unnecessarily fast to me, especially when checking two ports simultaneously.